packages/server/src/queue/queue.entity.ts

```diff
@@ -80,13 +66,7 @@ export class QueueModel extends BaseEntity {
   }
 
   async areThereOfficeHoursRightNow(now = new Date()): Promise<boolean> {
```

Comment:
I think we ought to just get rid of this method entirely and replace usages of the method with `isQueueOpen()` as appropriate